src/inference/dev_api/openvino/runtime/icore.hpp

```diff
@@ -117,12 +123,24 @@ class OPENVINO_RUNTIME_API ICore {
      * @param device_name Name of device load executable network on
      * @param config Optional map of pairs: (config parameter name, config parameter value) relevant only for this load
      * operation*
-     * @return An executable network reference
+     * @return An compiled model reference
      */
     virtual ov::SoPtr<ov::ICompiledModel> import_model(std::istream& model,
                                                        const std::string& device_name,
                                                        const ov::AnyMap& config = {}) const = 0;
 
+    /**
+     * @brief Creates an executable network from a previously exported network
+     * @param model model stream
+     * @param context Remote context
+     * @param config Optional map of pairs: (config parameter name, config parameter value) relevant only for this load
+     * operation*
+     * @return An compiled model reference
+     */
+    virtual ov::SoPtr<ov::ICompiledModel> import_model(std::istream& modelStream,
+                                                       const ov::RemoteContext& context,
+                                                       const ov::AnyMap& config) const = 0;
```

Comment:
Is default value missed?
```suggestion
                                                       const ov::AnyMap& config = {}) const = 0;
```

Reply:
It doesn't matter, it is an internal API which always called from the wrapper.